sig-docs/annual-report-2021.md

```diff
@@ -4,79 +4,72 @@
 
 1. What work did the SIG do this year that should be highlighted?
```

Comment:
any milestones hit or really amazing metrics to share about how much work y'all accomplished? example: page views, PRs reviewed, etc?

Reply:
did you work with other groups on thorny problems? examples of good collaboration, etc?

Reply:
I'll be highlighting the ongoing efforts for Dockershim that we flagged off last year in December.

Reply:
cc: @a-mccarthy for the page views/PRs reviewed bit, if you could please pull in the numbers?